config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
```

Comment:
this file will not be up to GIt, please
cp database.yml database.yml.example
git rm --cached config/database.yml
and in .gitignore add
/config/database.yml

Reply:
done

Reply:
please fix

Reply:
need to edit to sqlite2?

Reply:
you need ignore it?

Reply:
Sorry the question about ignoring is refer to the require "test" below

Reply:
I have created a new file and copy inside according to the instructions for database

Reply:
git commit -m "database deleted"
[init_project 5eb3601] database deleted
 2 files changed, 18 insertions(+), 1 deletion(-)
 create mode 100644 config/database.yml.example